src/shorts/shorts.sql.js

```diff
@@ -87,3 +87,10 @@ LEFT JOIN (
 ) comments ON s.shorts_id = comments.shorts_id
 WHERE s.tag REGEXP ?
 ORDER BY s.created_at DESC;`;
+
+// 쇼츠에 댓글 달기
+export const addComment = 
+`
+INSERT INTO COMMENT (shorts_id, user_id, comment, is_deleted, created_at)
+VALUES (?, ?, ?, 0, NOW());
```

Comment:
현재 데이터베이스 자체 설정에서 is_deleted는 false로 created_at은 insert 된 시각으로 들어가도록 되어 있어서 shorts_id, user_id, comment 값들만 세팅해줘도 됩니다! 아래와 같이 수정해도 된다는 말이었어요!

```suggestion
INSERT INTO COMMENT (shorts_id, user_id, comment)
VALUES (?, ?, ?);
```